client/ayon_max/mxp.py

```diff
@@ -0,0 +1,48 @@
+import os
+from ayon_core.lib import Logger
+
+
+def create_workspace_mxp(workdir, mxp_workspace=None):
+    dst_filepath = os.path.join(workdir, "workspace.mxp")
+    if os.path.exists(dst_filepath):
+        return
+
+    log = Logger.get_logger("create_workspace_mxp")
+    os.makedirs(workdir, exist_ok=True)
+    max_script = default_mxp_template()
+    if mxp_workspace and mxp_workspace.get("enabled_project_creation"):
+        max_script = mxp_workspace.get("mxp_workspace_script")
+        # Skip if mxp script in settings is empty
+        if not max_script:
+            log.debug("File 'workspace.mxp' not created. Settings value is empty.")
+            return
+
```

Comment:
```suggestion
    max_script = default_mxp_template()
    if mxp_workspace and mxp_workspace.get("enabled_project_creation"):
        max_script = mxp_workspace.get("mxp_workspace_script")
        # Skip if mxp script in settings is empty
        if not max_script:
            log.debug("File 'workspace.mxp' not created. Settings value is empty.")
            return

    os.makedirs(workdir, exist_ok=True)
```
Let's create the fodlers only if we need to..
Also, is this logic correct.

If `mxp_workspace.get("enabled_project_creation")` is False then it will still write the `mxp` but then the default one?